### PR TITLE
KDP: Update documentation for consuming services

### PR DESCRIPTION
**Pull request by embik**
KDP has evolved a bit, so this updates the documentation with some screenshots and updated information around service consumption.

### CODE REVIEW

File: content/developer-platform/platform-users/consuming-services/_index.en.md [hunk 1]
```diff
@@ -20,10 +20,9 @@ ultimately translated into a kcp `APIExport` with a number of `APIResourceSchema
 
 ## Browsing Services
 
-Login to the KDP Dashboard and choose your organization. Then select "Services" in the menu bar to
-see a list of all available Services. This page also allows to create new services, which is
-further described in [Your First Service]({{< relref "../../tutorials/your-first-service" >}}) for
-service owners.
+Log in to the KDP Dashboard and choose your organization. Then select "Services Catalog" in the menu bar to see a list of all Services that you have access to. For each service that hasn't been added to the organization yet a button is offered to add it to the organization.
```
Comment (xrstf) [inline content/developer-platform/platform-users/consuming-services/_index.en.md:23]: "Service" or "service" ?

Comment (embik) [reply content/developer-platform/platform-users/consuming-services/_index.en.md]: "service".

File: content/developer-platform/platform-users/consuming-services/_index.en.md [hunk 2]
```diff
@@ -32,13 +31,18 @@ Note that every Service shows:
...
+After clicking on "Add to Organization" in the service catalog, the following wizard will open to confirm adding a service:
+
+![Add Service Wizard](add-service-wizard.png?classes=shadow,border&height=200 "Wizard for adding a service to the workspace, showing options for creating default rules and resource claims")
 
-### Dashboard
+Several options are offered when adding a service:
 
-(TODO: currently the UI has no support for this.)
+- "Create default roles for resources managed by this service" will optionally create a standard set of RBAC roles within the workspace that allow giving different levels of access to this specific service. This is not mandatory, but otherwise roles need to be created yourself.
+- The "Resources" section informs you about resources that the service is requesting access (claiming) to function properly, e.g. to be able to read or write secret data associated with objects of that service. Service owners are encouraged to gracefully handle situations in which resource access is not granted, but service quality and functionality will likely suffer.
```
Comment (xrstf) [inline content/developer-platform/platform-users/consuming-services/_index.en.md:45]: You could add "you can grant/revoke access to resources for a service at any time later" or similar.

Comment (embik) [reply content/developer-platform/platform-users/consuming-services/_index.en.md]: :+1:

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**